sources/utils.go

```diff
@@ -43,3 +46,9 @@ func HealthyHealthCheckResult(checkType health.CheckType) health.HealthCheckResu
 		State: health.New_HealthState(health.HealthState_HEALTHY),
 	}
 }
+
+func ErrorToRichParamsMap(err error) map[string]interface{} {
+	return map[string]interface{}{
+		"richErrorMessage": werror.GenerateErrorString(err, false),
```

Comment:
I might call this `stacktrace` to match the svc1log field since the content is similar

Reply:
Per our discussion, adjusted to just use the safe params of the error